mdpow/tests/test_solvation.py

```diff
@@ -20,10 +20,10 @@
 
 @pytest.fixture
 def setup(tmpdir):
+    os.system('echo')
```

Comment:
remove

Reply:
done, I think that was an artifact from troubleshooting gromacs not loading.